### ISSUE TITLE
Crash on Fresh Install (mono)

### ISSUE THREAD

**Issue by Allcharles** [QutEcoacoustics/audio-analysis#225]
## Describe the bug
Download of AP using download_ap.ps1 fails on a clean install of Linux Mint 18.3. When attempting to run AnalysisPrograms.exe the system crashes before any commands can be run.

## Expected behavior 
Should be able to use AP through mono without issue.

## Diagnostics
- [Terminal Log](https://github.com/QutEcoacoustics/audio-analysis/files/2950620/terminalOutput.txt)
- OS: Linux Mint 18.3 (Sylvia)
- AP.exe version: 19.3.0.5 (Cannot confirm, this is output from powershell download)
- Mono: 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)
- Powershell scripts used, `script_ap.ps1` runs `download_ap.ps1`. [Scripts](https://github.com/QutEcoacoustics/audio-analysis/files/2950632/powershell.commands.tar.gz)

**Comment by atruskie** [comment]
I think the problem here is your mono version. We require Mono 5.12 or higher. Can you update it and see if that does the trick?

**Comment by Allcharles** [comment]
download_ap.ps1 no longer works on Linux Mint. It now gives the following error message:
```
Get-Command : The term 'curl.exe' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At /home/charles/Documents/GitHub/ap-desktop/ap/download_ap.ps1:189 char:13
+     $curl = Get-Command curl, curl.exe -CommandType Application | sel ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (curl.exe:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
```

**Comment by Allcharles** [comment]
I have updated to Mono to 5.18.0.268, and am running the newest AP version 19.2.2.1. Unfortunately it still crashes, same error output with and without log level 7.
[log](https://github.com/QutEcoacoustics/audio-analysis/files/2951147/log_20190311T075210Z.txt)
[terminal](https://github.com/QutEcoacoustics/audio-analysis/files/2951159/terminal.txt)

**Comment by Allcharles** [comment]
Hopefully final comment on download_ap.ps1. New commit broke it from running correctly on windows PowerShell as well. I tested the command outside the script in a PowerShell terminal and it returns the same error.

Windows 10 Pro Version 10.0.17134 Build 17134
PowerShell Version 5.1.17134.590
download_ap.ps1 [Version](https://github.com/QutEcoacoustics/audio-analysis/commit/b6a34a5331a04bcb11dac1a97cc72137437c1bf5)
Fails on line 207 (`if (Get-Command unzip -CommandType Application) {`)
Error:
```
Get-Command : The term 'unzip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At D:\Documents\GitHub\VRES\ap\download_ap.ps1:207 char:9
+     if (Get-Command unzip -CommandType Application) {
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (unzip:String) [Get-Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
```

**Comment by atruskie** [comment]
A new version of download_ap.ps1 here 3d0d2848f922156717b2028143860a22a4022963

---

I can't reproduce this error (using mono 5.18)

I've taken a wild stab at a fix in e3317bd4dfcee59bdc257759a8e8aa905cb19da3. 

Please test again, make sure you download the artefact from AppVeyor.

**Comment by atruskie** [comment]
@Allcharles your log indicates that you are not using the latest version (build 686 from AppVeyor).

Here is a direct link: https://ci.appveyor.com/project/QUTEcoacousticsResearchGroup/audio-analysis/builds/22996165/artifacts
